core/src/i18n/pl.yaml

```diff
@@ -112,6 +107,11 @@ observability:
   title: Obserwowalność
 pods:
   title: Pody
+  restarts: Ponownych uruchomień
+  node: Węzeł
+deployments:
+  title: Deploymenty
+service-catalog:
 replica-sets:
   title: Zestawy replik
```

Comment:
Are we sure we want to have `DaemonSety` but `Zestawy Replik`?
Shouldn't we unify it and have both {`DaemonSety` and `ReplicaSety`} or {`Zestawy demonów` and `Zestawy replik`}?